util/MeasureCalculation.ts

```diff
@@ -0,0 +1,34 @@
+import { Calculator, CalculatorTypes } from 'fqm-execution';
+import { TestCaseInfo } from '../state/atoms/patientTestCase';
+import { createPatientBundle } from './fhir';
+
+/**
+ * Takes in a patient's test case info, a measure bundle, and a measurement
+ * period start and end and returns a measureReport
+ * passed in filename and fileContents
+ * @param patientTestCase {TestCaseInfo}
+ * @param mb {fhir4.Bundle}
+ * @param mpStart {string}
+ * @param mpEnd {string}
+ * @returns MeasureReport
```

Comment:
This is opinionated, but one of the reasons TypeScript is so good is that it basically eliminates the need for JSDoc comments like this. All of this info is clearly identifiable by just looking at the function signature, so my opinion is that we shouldn't even put waste time writing these.